lomap/mcs.py

```diff
@@ -526,37 +512,37 @@ def set_ring_counter(mol):
         Chem.SanitizeMol(self.molj)
 
         # Set chirality flags from 3D coords if working in 3D
-        if self.options.threed:
+        if threed:
             Chem.rdmolops.AssignAtomChiralTagsFromStructure(self.moli,replaceExistingTags=True)
             Chem.rdmolops.AssignAtomChiralTagsFromStructure(self.molj,replaceExistingTags=True)
 
-        if not options.verbose == 'pedantic':
+        if not verbose == 'pedantic':
             lg = RDLogger.logger()
             lg.setLevel(RDLogger.CRITICAL)
 
         # Local pointers to the passed molecules without hydrogens
         # These variables are defined as private
         try:
-            self.__moli_noh = AllChem.RemoveHs(moli)
-            self.__molj_noh = AllChem.RemoveHs(molj)
+            self._moli_noh = AllChem.RemoveHs(moli)
+            self._molj_noh = AllChem.RemoveHs(molj)
         except Exception:
-            self.__moli_noh = AllChem.RemoveHs(moli, sanitize=False)
-            self.__molj_noh = AllChem.RemoveHs(molj, sanitize=False)
+            self._moli_noh = AllChem.RemoveHs(moli, sanitize=False)
+            self._molj_noh = AllChem.RemoveHs(molj, sanitize=False)
 
-            Chem.SanitizeMol(self.__moli_noh, sanitizeOps=Chem.SanitizeFlags.SANITIZE_SETAROMATICITY)
-            Chem.SanitizeMol(self.__molj_noh, sanitizeOps=Chem.SanitizeFlags.SANITIZE_SETAROMATICITY)
+            Chem.SanitizeMol(self._moli_noh, sanitizeOps=Chem.SanitizeFlags.SANITIZE_SETAROMATICITY)
+            Chem.SanitizeMol(self._molj_noh, sanitizeOps=Chem.SanitizeFlags.SANITIZE_SETAROMATICITY)
 
         # MCS calculation. In RDKit the MCS is a smart string. Ring atoms are
         # always mapped in ring atoms.
         # Don't add the mcs result as a member variable as it can't be pickled
-        __mcs = rdFMCS.FindMCS([self.__moli_noh, self.__molj_noh],
-                                    timeout=options.time,
-                                    atomCompare=rdFMCS.AtomCompare.CompareAny,
-                                    bondCompare=rdFMCS.BondCompare.CompareAny,
-                                    matchValences=False,
-                                    ringMatchesRingOnly=True,
-                                    completeRingsOnly=True,
-                                    matchChiralTag=False)
+        __mcs = rdFMCS.FindMCS([self._moli_noh, self._molj_noh],
```

Comment:
`__mcs` still has dunderscore here, is that intentional?

Reply:
The class dunderscores do that annoying name mangling in Python, this isn't a class attribute so it's less egregious. Just walking the line between keeping the diff minimal and making the code more usable.